snapshot/cli/cli.go

```diff
@@ -184,31 +185,44 @@ func (c *ingestGitCommitCommand) run(db snapshot.DB, _ *cobra.Command, _ []strin
 //	--basis="<rev>" --ref="master" --ttl="336h" --bundlestore_url="http://localhost:9094/bundle"
 // Stdout: "http://localhost:9094/bundle/bs-<rev>-master.bundle"
 type createGitBundleCommand struct {
-	file  string
-	basis string
-	ref   string
-	ttld  time.Duration
-	keep  bool
+	basis  string
+	ref    string
+	ttld   time.Duration
+	output string
 }
 
+const outputTypeLocation = "location"
+const outputTypeSnapshotID = "snapshot-id"
+
+var outputTypes = []string{outputTypeLocation, outputTypeSnapshotID}
+
 func (c *createGitBundleCommand) register() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "publish_git_bundle",
 		Short: "Creates and uploads a git bundle file as specified by basis/ref for the cwd repo",
 	}
 	// See https://git-scm.com/docs/git-bundle for basis, ref usage
-	cmd.Flags().StringVar(&c.file, "file", "", "Output bundle file (defaults to temp file)")
 	cmd.Flags().StringVar(&c.basis, "basis", "", "Basis for bundle")
 	cmd.Flags().StringVar(&c.ref, "ref", "master", "Reference to be packaged")
 	cmd.Flags().DurationVar(&c.ttld, "ttl", bundlestore.DefaultTTL, "Stored bundle TTL (duration from now)")
-	cmd.Flags().BoolVar(&c.keep, "keepFile", false, "Keep created bundles file")
+	cmd.Flags().StringVar(&c.output, "output", "location", fmt.Sprintf("Output type, one of: %q", outputTypes))
 	return cmd
 }
 
 func (c *createGitBundleCommand) run(db snapshot.DB, _ *cobra.Command, _ []string) error {
```

Comment:
To help remove some tech debt, while you understand this section of code can you add a comment above run() explaining what it is doing?  (see example below - note: I may have it wrong)
/*
1. validate the input parameters
2. create the bundle name (snapshot id?) from the ?git sha?
3. create the bundle
4. upload the bundle
5. return the bundle's snapshot id or upload location (depending on the requested responseType)
*/